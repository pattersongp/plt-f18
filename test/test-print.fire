func int main = () => {
    print(100);
    int tmp = 50; 
    print(tmp);
    printer(150);
    return 0;
}

func void printer = (int x ) => {
    print(x);
}
