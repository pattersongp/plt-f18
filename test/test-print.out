100
50
150
